setup.py

```diff
@@ -10,7 +10,7 @@
       author_email="bmhatfield@gmail.com",
       url="https://github.com/bmhatfield/riemann-sumd",
       package_dir={'': 'lib'},
-      py_modules=['event', 'loader', 'scheduler', 'sender', 'task'],
+      py_modules=['event', 'loader', 'scheduler', 'sender', 'task', 'task_runner'],
```

Comment:
Can we call the module just `runner` to keep with the nice single-word name theme?